src/Breadcrumb/readme.md

```diff
@@ -17,46 +17,69 @@ Type: UI Component
 
 ## Element information
 
-This React component is based on SEB Bootstrap style. Supports customization and configurations. The component name is `Breadcrumb` and the selector is `<Breadcrumb/>`.
+This React component is based on SEB Bootstrap style. Supports customization and configurations. The component name is `Breadcrumb` and the selector is `<Breadcrumb/>`. A supplimentary child component is also available for use `BreadcrumbItem`. See the examples below.
 
 ## Basic use
 
+It can be used in these combinations
 ```html
-<Breadcrumb list="{breadcrumbListObj}" onClick="{clickHandler}" />
+<!-- Passing the list and one handler for all link clicks -->
+<Breadcrumb list={breadcrumbList} onNavigate={commonHandler} />
+
+<!-- Rendering a list of BreadcrumbItem -->
+<Breadcrumb>
+    {breadcrumbList.map((item: BreadcrumbItemProps, i: number) => {
+        /** Do whatever logic before rendering */
+        return (
+            <BreadcrumbItem key={i} {...item} onNavigate={individualHandler}/>
+        )
+    })}
+</Breadcrumb>
+
+<!-- Rendering a bunch BreadcrumbItem individually -->
+<Breadcrumb>
+    <BreadcrumbItem href="#/"><HomeIcon/></BreadcrumbItem>
+    <BreadcrumbItem href="#/user">User</BreadcrumbItem>
+    <BreadcrumbItem href="#/edit">Edit</BreadcrumbItem>
+</Breadcrumb>
 ```
+
+**onNavigate** handler can be passed to the parent as common handler, or passed to each individual child as a unique handler for each item
+
 ```typescript
 /** If you use hash router, you need to pass the hash in the href */
 const breadcrumbListObj: Array<BreadcrumbItem> = [
-    { text: "Home", href: "#/" },
-    { text: "Second page", href: "#/second-page" }
+    { children: "Home", href: "#/" },
```

Comment:
again very confusing to have a property called children which makes it sound like it's an array or some kind and then figuring out it's a string.

Reply:
I'm defaulting back to the native HTML. The preferred way of adding breadcrumbs is this:
```tsx
<Breadcrumb>
    <BreadcrumbItem>Home</BreadcrumbItem>
    <BreadcrumbItem>Other page</BreadcrumbItem>
</Breadcrumb>
```
But if you chose to generate a list, then you can still tap into the `children` prop, just like you do with normal JS/HTML setup

Reply:
So in this case the children in a list  can be anything including just a string?

Reply:
yes